frontend/src/metabase-types/api/dashboard.ts

```diff
@@ -1,5 +1,28 @@
+import {
+  ParameterTarget,
+  ParameterId,
+  Parameter,
+} from "metabase-types/types/Parameter";
+import { CardId, SavedCard } from "metabase-types/types/Card";
+
 export interface Dashboard {
   id: number;
   name: string;
   model?: string;
+  ordered_cards: DashboardOrderedCard[];
+  parameters: Parameter[];
```

Comment:
just adding the properties I need, for now. We can fill out the rest of the type info later.